pkg/services/ngalert/schedule/schedule.go

```diff
@@ -306,6 +310,7 @@ func (sch *schedule) processTick(ctx context.Context, dispatcherGroup *errgroup.
 		}
 
 		if isReadyToRun {
+			sch.log.Error("\n\nRule is ready to run", "uid", item.UID, "tick", tickNum, "frequency", itemFrequency, "offset", offset) // TODO: Remove me!
```

Comment:
Don't forget to remove me 😄

Reply:
I think I would change it to debug - we need to be able to understand what was the calculated offset if we ever have a need to debug this.

Reply:
Thanks for highlighting this, forgetting to remove debug logs is my achilles heel :laughing: